diff --git a/target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6000-re-ss-01.dts b/target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6000-re-ss-01.dts
index b2f35fd..dfdc344 100644
--- a/target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6000-re-ss-01.dts
+++ b/target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6000-re-ss-01.dts
@@ -12,15 +12,15 @@
 #include <dt-bindings/leds/common.h>
 
 / {
-	model = "JDCloud RE-SS-01";
+	model = "JDCloud AX1800 Pro";
 	compatible = "jdcloud,re-ss-01", "qcom,ipq6018";
 
 	aliases {
 		serial0 = &blsp1_uart3;
 
-		led-boot = &led_status_blue;
+		led-boot = &led_status_green;
 		led-failsafe = &led_status_red;
-		led-running = &led_status_green;
+		led-running = &led_status_blue;
 		led-upgrade = &led_status_red;
 
 		ethernet1 = &dp2;
diff --git a/target/linux/qualcommax/image/ipq60xx.mk b/target/linux/qualcommax/image/ipq60xx.mk
index d810562..9be7d80 100644
--- a/target/linux/qualcommax/image/ipq60xx.mk
+++ b/target/linux/qualcommax/image/ipq60xx.mk
@@ -325,7 +325,7 @@ define Device/jdcloud_re-ss-01
 	$(call Device/FitImage)
 	$(call Device/EmmcImage)
 	DEVICE_VENDOR := JDCloud
-	DEVICE_MODEL := RE-SS-01
+	DEVICE_MODEL := AX1800 Pro
 	KERNEL_SIZE := 6144k
 	BLOCKSIZE := 128k
 	SOC := ipq6000
